<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<arg name="obs1" default="SQ01s"/>

    <node ns="$(arg obs1)" name="obstacle" pkg="puma" type="obstacle.py" output="screen" clear_params="true">
        <rosparam file="$(find puma)/param/obstacle1.yaml" command="load"/>
    </node>

    <node ns="$(arg obs1)" name="panther_obs_commands" pkg="puma" type="panther_obs_commands.py" output="screen"/>

    <node ns="$(arg obs1)" pkg="rqt_gui" type="rqt_gui" name="obs_rqt_gui" args="--perspective-file $(find puma)/rqt_cfgs/obs_demo_interface.perspective"/>

    <node ns="$(arg obs1)" pkg="behavior_selector" type="behavior_selector_node.py" name="obs_behavior_selector" output="screen" />

</launch>